### ISSUE TITLE
How to start with devtools?

### ISSUE THREAD

**Issue by marekn22** [emmenko/redux-react-router-async-example#49]
for now I just changed webpack.config.js
'**DEVTOOLS**': process.env.DEVTOOLS === 'true' ? true : true
and then 'npm start' shows devtools

**Comment by emmenko** [comment]
There is already a script for that ;)

```
npm run start:devtools
```

**Comment by marekn22** [comment]
I should mention that I tried it. I got the following error:
....
'DEVTOOLS' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\
node_modules\npm\bin\npm-cli.js" "run" "start:devtools"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! redux-react-router-async-example@0.0.1 start:devtools: `npm run build &
& DEVTOOLS=true node server.js`
npm ERR! Exit status 1
